index.js

```diff
@@ -8,9 +8,13 @@ const pluginDefaults = {
     'default-src': ['https:', 'unsafe-inline', 'unsafe-eval'],
     'report-uri': '/csp_reports'
   },
-  headerKey: 'Content-Security-Policy-Report-Only'
+  headerKey: 'Content-Security-Policy-Report-Only',
```

Comment:
should we change this to reportOnly: true? and if it's false, then have the key be `content-security-policy`?